Arrays/3443MaximumManhattanDistanceAfterKChanges.py

```
@@ -0,0 +1,106 @@
+import sys
+class Solution:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)